resources/graphics.py

```diff
@@ -33,6 +33,7 @@
 				"pointerAdd",
 				"pointerRemove",
 				"pointerRotate",
+                "pointerPivot",
```

Comment:
We've ended up with spaces here.

Reply:
That's no good! Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/5558/commits/18b122489faf389ee7ab5dcad65544b4d7b70021